autoload/wintabs/ui.vim

```diff
@@ -165,6 +165,37 @@ function! s:truncate_line(window, bufline, width)
   return line
 endfunction
 
+function! s:get_tab_name(n)
+  " show number of tabs
+  if g:wintabs_ui_show_vimtab_name == 0
+    return a:n
+  endif
+
+  " getting tab name
+  let title = ''
+  let s:taboo = get(g:, 'loaded_taboo', 0)
+  if s:taboo
+    let title = TabooTabTitle(a:n)
+  endif
+
+  if empty(title) && exists('*gettabvar')
+    let title = gettabvar(a:n, 'title')
+  endif
+
+  if empty(title)
+    let buflist = tabpagebuflist(a:n)
+    let winnr = tabpagewinnr(a:n)
+    let title = bufname(buflist[winnr - 1])
+    let title = split(title, "/")[-1]
```

Comment:
This can error if you simply issue an `:tabnew` as the new vimtab has no buffers and `title` is empty.